recipes/libdwarf/all/conanfile.py

```diff
@@ -20,11 +21,13 @@ class LibdwarfConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "with_dwarfgen": [True, False],
+        "with_dwarfdump": [True, False],
```

Comment:
You'll need to update the license handling because dwarfdump is GPL: https://www.prevanders.net/dwarflicense.html

Add the GPL variant it uses to the `licenses` attribute.

```
license = ("LGPL-2.1-only", "GPL-2.0-only", "BSD-2-Clause-Views")
```

Then configure the license in the `configure` method so that it only includes the GPL license when `with_dwarfdump` is enabled.

Additionally, you'll want to double check the project's supplied GPL license is copied to the `licenses` directory.

Reply:
Happy to do so

Reply:
Thanks, much appreciated!